msg/TeamData.msg

```diff
@@ -7,59 +7,42 @@ std_msgs/Header header
 
 # Every value is an array because we can have multiple robots communicating with us.
 # The values match with the robot ids
```

Comment:
I assume, these comments are now obsolete...

Reply:
The teamData message will be handled separately depending on the new protocol.